changelog.json

```diff
@@ -920,15 +942,15 @@
       "[Fixed] Undo cherry-picking onto a remote branch now works as expected - #11921",
       "[Fixed] Dragging commits onto current branch no longer results in a broken state - #11915",
       "[Fixed] Get correct commit summary info in rebase - #11853",
-      "[Improved] Cancel drag operation with escape key - #11925",
+      "[Improved] Cancel drag operation with  target key - #11925",
```

Comment:
These replacements of `escape` with `target` are curious. I am not sure where they originated from.. but sure enough looks like it was on my release commit... had to be some sort of automatic thing as I have not done a find/replace recently.